stl/inc/chrono

```diff
@@ -5892,6 +5911,21 @@ namespace chrono {
                 }
                 _Os << _Time.tm_mday;
                 return true;
+            case 'j':
+                if constexpr (_Is_specialization_v<_Ty, duration>) {
+                    _Os << _STD abs(_CHRONO round<days>(_Val).count());

```

Comment:
I believe that these `round` calls will print `days{4} + 23h` as `"5"` for `%j`. We should have test coverage for positive and negative cases where rounding behaves differently from truncation.